### ISSUE TITLE
Export to FINN-ONNX fails: 'QuantHardTanh' object has no attribute 'quant_output_scale'

### ISSUE THREAD

**Issue by surajkarki66** [sefaburakokcu/finn-quantized-yolo#10]
## 🐛 Bug Report

I'm attempting to export a quantized YOLOv1 tiny quant model to FINN-ONNX using Brevitas and encountered the following critical error:
I am currently trying to use latest brevitas version for QAT.

### 📋 Environment

- **OS:** Windows
- **PyTorch version:** 2.8.0+cpu
- **Brevitas version:** latest from `pip`
- **FINN version:** latest from `main`
- **Python version:** 3.10

---

### ⚠️ Observations

- The error is raised from a call to `.quant_output_scale()` on a `QuantHardTanh` layer.

Command Prompt Output:

C:\Users\Leica-Admin\Documents\quantized-yolov5\.venv\lib\site-packages\brevitas\graph\equalize.py:69: UserWarning: fast_hadamard_transform package not found, using standard pytorch kernels
  warnings.warn("fast_hadamard_transform package not found, using standard pytorch kernels")
export: data=data/hpd2_modified.yaml, weights=runs/train/exp/weights/best.pt, imgsz=[416, 416], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=11, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['finn_onnx'], nodetect=True
YOLOv5  9b696a4 torch 2.8.0+cpu CPU

  0                -1  1       233  models.common.QuantConv                 [3, 8, 3, 1, None, 1, True, True, 8, 4]
  1                -1  1         0  torch.nn.modules.pooling.MaxPool2d      [2, 2, 0]
  2                -1  1       593  models.common.QuantConv                 [8, 8, 3, 1, None, 1, True, True, 4, 4]
  3                -1  1         0  torch.nn.modules.pooling.MaxPool2d      [2, 2, 0]
  4                -1  1      1185  models.common.QuantConv                 [8, 16, 3, 1, None, 1, True, True, 4, 4]
  5                -1  1         0  torch.nn.modules.pooling.MaxPool2d      [2, 2, 0]
  6                -1  1      4673  models.common.QuantConv                 [16, 32, 3, 1, None, 1, True, True, 4, 4]
  7                -1  1         0  torch.nn.modules.pooling.MaxPool2d      [2, 2, 0]
  8                -1  1     16241  models.common.QuantConv                 [32, 56, 3, 1, None, 1, True, True, 4, 4]
  9                -1  1         0  torch.nn.modules.pooling.MaxPool2d      [2, 2, 0]
 10                -1  1     52625  models.common.QuantConv                 [56, 104, 3, 1, None, 1, True, True, 4, 4]
 11                -1  1    195105  models.common.QuantConv                 [104, 208, 3, 1, None, 1, True, True, 4, 4]
 12                -1  1     11761  models.common.QuantConv                 [208, 56, 1, 1, None, 1, True, True, 4, 4]
 13                -1  1     52625  models.common.QuantConv                 [56, 104, 3, 1, None, 1, True, True, 4, 4]
 14                -1  1     16848  models.common.QuantSimpleConv           [104, 18, 3, 1, None, 1, 8, 4, True]
 15              [14]  1       342  models.yolo.Detect                      [1, [[10.0, 14.0, 23.0, 27.0, 37.0, 58.0]], [18], False, True]
C:\Users\Leica-Admin\Documents\quantized-yolov5\.venv\lib\site-packages\torch\_tensor.py:1645: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\pytorch\c10/core/TensorImpl.h:1942.)
  return super().rename(names)
Reversing anchor order
C:\Users\Leica-Admin\Documents\quantized-yolov5\.venv\lib\site-packages\torch\functional.py:554: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\pytorch\aten\src\ATen\native\TensorShape.cpp:4324.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Model Summary: 800 layers, 352231 parameters, 352231 gradients, 0.0 GFLOPs

Fusing layers...
Model Summary: 800 layers, 352231 parameters, 352231 gradients, 0.0 GFLOPs

PyTorch: starting from runs\train\exp\weights\best.pt (0.8 MB)

FinnOnnx: starting export with torch 2.8.0+cpu...
FinnOnnx: export failure: 'QuantHardTanh' object has no attribute 'quant_output_scale'

Export complete (2.42s)
Results saved to C:\Users\Leica-Admin\Documents\quantized-yolov5\runs\train\exp\weights
Visualize with https://netron.app

**Comment by surajkarki66** [comment]
Also one question. How important is scale.npy file? I mean this error came while getting the scale value of HardQuantTanh.

Please help.
Thank You in advance.
